doc/vim-pencil.txt

```diff
@@ -0,0 +1,651 @@
+*vim-pencil.txt*  Rethinking Vim as a tool for writers  Last change: 2020-07-29
+
+      _     _  _        ____  _____ _      ____  _  _    
+     / \ |\/ \/ \__/|  /  __\/  __// \  /|/   _\/ \/ \   
+     | | //| || |\ /|  | |  \/  |  |  \ |||  |  | || |   
+     | \// | || |  ||  |  __/|  /_ | | \|||  \_ | || |_/\
+     \__/  \_/\_/  \|  \_/   \____\\_/  \|\____/\_/\____/
+
+            Rethinking Vim as a tool for writers 
+
+CONTENTS                                                             *vimwiki*
```

Comment:
Vimwiki? This looks like a copy/paste leftover from something you used as a template perhaps?